public-html/asset/scss/components/form.scss

```diff
@@ -1,112 +1,126 @@
 .container {
-	max-width: 380px;
-	margin: 7% auto;
+  max-width: 380px;
+  margin: 7% auto;
 }
 
 .header {
-	text-align: center;
+  text-align: center;
 }
 
 .header h1 {
```

Comment:
one selector per line ( check at all the places )

Reply:
fixed